src/java/picard/analysis/directed/CollectHsMetrics.java

```diff
@@ -45,15 +45,76 @@
+            "" +
+            "Both CollectTargetedPCRMetrics and CalculateHybridSelection metrics share virtually identical program structures except " +
+            "for the name of their targeting mechanisms (e.g. bait set or amplicon set).  Both depend on the TargetMetricsCollector " +
+            "and collect metrics on all reads in the INPUT SAM/BAM file. <br /><br /> " +
+            "" +
+            "Hybrid selection (HS) enables targeted sequencing analysis via the capture of specified genomic DNA " +
+            "sequences (doi:10.1038/nbt.1523).  It is commonly used to characterize exon sequences from genomic DNA " +
+            "or filter out bacterial DNA sequences from clinical samples.  <br /><br /> " +
+            "" +
+            "The technique involves the capture of unique regions of genomic DNA (targets) using synthetic RNA baits." +
+            "   The baits are synthesized with biotinylated nucleotides to facilitate capture of bait:target hybrids on" +
+            " streptavidin beads.  The captured target sequences are amplified, sequenced, and processed for variant calling. <br /><br />" +
+            "" +
+            "This tool requires an aligned SAM or BAM file as well as bait and target interval files.  " +
+            "The bait and target files can be formatted as VCF or as interval_list files.  For information on VCF files, please see:" +
+            "<br /><br />www.1000genomes.org/wiki/analysis/variant%20call%20format/vcf-variant-call-format-version-41" +
+            "" +
+            "<br /><br />This tool provides multiple outputs including:" +
+            "<li>The numbers of aligned reads that pass the sequencing vendor's quality \"Chastity\" filter (PF)* " +
+            "<li>The numbers of reads and their constituent bases that map to the reference sequence " +
+            "<li>The per read and per base coverage depth for both bait and target intervals<br /><br />" +
+            "The tool also calculates hybrid selection penalty values, defined as the cost incurred to get 80% of target bases" +
+            " to a specific amount of depth coverage.  For example, if a design consists of 10 megabases of target and" +
+            " 10X coverage depth is required, then the amount of sequencing needed is the PF_ALIGNED_BASES = 10^7 * 10 * HS_PENALTY_10X.<br /><br />" +
+            "" +
+            "Both AT_DROPOUT and GC_DROPOUT metrics are indicated, which show the percentage of " +
+            "reads dropped from an analysis due to the inability to map to the reference as result of excessively " +
+            "AT- or GC-rich regions respectively.<br /><br />" +
+
+            "The PER_TARGET_COVERAGE option can be used to output GC content and mean sequence depth information for every target interval." +
+            "<br /><br />" +
+            "" +
+            "Please note that coverage depth at each locus should not exceed a limit of java MAX_SHORT ~32K.  This is because the " +
```

Comment:
the coverage depth metric is not using int[] not short[] (in hsmetrics and targetedmetrics